plutus-core/cost-model/test/TestCostModels.hs

```diff
@@ -117,6 +117,10 @@ prop_sha3 :: Property
 prop_sha3 =
     testPredictOne sHA3 (getConst . paramSHA3)
 
+prop_blake2b :: Property
+prop_blake2b =
+    testPredictOne bLAKE2b (getConst . paramBlake2b)
+
```

Comment:
This will make `plutus-core:cost-model-test` fail because it's trying to read non-existent R data.  That probably doesn't matter though: this doesn't get run in CI (the cabal file says it's a benchmark) and we will want this once we've added benchmarks for `Blake2b`.